README.md

```diff
@@ -24,30 +25,42 @@ These features are not implemented in this repo, but in Neovim core. See `:help
 * Requires [Neovim v0.5.0](https://github.com/neovim/neovim/releases/tag/v0.5.0) or [Nightly](https://github.com/neovim/neovim/releases/tag/nightly). Update Neovim and nvim-lspconfig before reporting an issue.
 
 * Install nvim-lspconfig like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
-```vim
-Plug 'neovim/nvim-lspconfig'
-```
+
```

Comment:
why are these blank lines needed or desirable? there is already plenty of whitespace / visual separation. Too much spacing reduces density far too much.

Reply:
markdown code convention insists on keeping empty lines between blocks. even though, there were some of these empty lines, but not everywhere